tests/docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '2.1'
+version: '3'
```

Comment:
Why is this? 3rd version does not support some features that 2nd does. They are not used in this particular file, but still.